### PR TITLE
Add generic ButtonBulkAction and javascript to handle open in tabs

### LINKED ISSUE #16455: [Order List] Bulk action to open multiple orders in new tabs
**Is your feature request related to a problem? Please describe.**
To open multiple orders in a new tab with the bulk actions.
**Describe the solution you'd like**
Add "Open in new tabs" in the bulk action list. 

If the browser blocks the opening of commands in other tabs, a modal will be displayed to the user to explain how to solve the problem.

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add generic ButtonBulkAction and javascript to handle open in tabs
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16455
| How to test?  | In order view select some orders and use the bulk action `Open in new tabs` By default your browser should block and only open one tab Then in the list page you will see an alert advising you to change your browser settings to allow more popup Once it is done you can open as many tabs as you want

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16552)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      let handle = window.open(this.router.generate(route, routeParams));
+      if (handle) {
+        handle.blur();
+        window.focus();
+      } else {
+        allTabsOpened = false;
+      }
+
+      if (!allTabsOpened) {
+        alert('It looks like you have exceeded the number of tabs allowed. Check your browser settings to open multiple tabs.');
```
Comment (matthieu-rolland) [inline admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js]: Maybe this message could be translated? (by using the translator from the template, storing the message in an html element)

File: admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import Router from '../../router';
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js]: Move this after the license block please

File: admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js (hunk 1 above)
Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js]: can be store like  `.data('route')` for example

File: admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js (hunk 2 above)
Comment (jolelievre) [reply admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js]: Oooohhh so this is where it was hidden!! Damn PHPStorm I didn't understand where the import came from ^^

File: admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js (hunk 1 above)
Comment (jolelievre) [reply admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js]: Yep, that's what I planned but I forgot to add it Thanks guys

**Comment by jolelievre** [comment]
@PierreRambaud @matthieu-rolland I forgot to push it yesterday, but it should be ok now

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]
Rebased :ballot_box_with_check:

**Comment by Progi1984** [comment]
Thanks @jolelievre

**Pull request merged**